lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -8,57 +8,90 @@
 @import '../../scss-shared/nx-text-helpers';
 @import '../../scss-shared/nx-container-helpers';
 
+$nx-dropdown-width: 250px;
+
 .nx-dropdown {
   display: inline-block;
   position: relative;
+  width: $nx-dropdown-width;
 }
 
 .nx-dropdown__toggle {
+  display: flex;
   margin: 0;
   outline: none;
+  text-align: left;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+  width: $nx-dropdown-width;
+
+  .nx-dropdown__toggle-label {
+    flex-grow: 1;
```

Comment:
I don't know that it's any better or worse, but another way to do this would be to put `margin-right: auto;` on this element instead

Reply:
> I don't know that it's any better or worse, but another way to do this would be to put `margin-right: auto;` on this element instead

I feel like `flex-grow` is more obvious in its intent? I guess, for those who know flex.